inference/app/static/1.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
?

Reply:
will remove them just added for reference